### DISCUSSION TITLE
How do I load an ohmyzsh/lib/*.zsh file?

### DISCUSSION THREAD

**Discussion by tbenthompson** [zdharma-continuum/zinit#363] [Q&A] [answered]
For example:

https://github.com/ohmyzsh/ohmyzsh/blob/d87ab251c7fe18626b2d0c4e4a184e7bed7c508b/lib/git.zsh

contains the `git_current_branch()` function which is used by the git plugin.

Currently, I have in my `.zshrc`:
```
zinit snippet OMZP::git
```
This load everything in the [git plugin here](https://github.com/ohmyzsh/ohmyzsh/blob/master/plugins/git/git.plugin.zsh), but that depends on the `git.zsh` file linked above.

Since I figured this might be a "lib", I've also tried:
```
zinit snippet OMZL::git
```
but that failed with:
```
Setting up snippet: OMZL::git
Downloading `OMZL::git` (with curl, wget, lftp)…
curl: (22) The requested URL returned error: 404                                                                                              
curl: (22) The requested URL returned error: 404       
```

Anyone have suggestions?

**Comment by sm1999** [comment, marked as answer]
Use the actual filename e.g.`zinit snippet OMZL::git.zsh`

**Reply by tbenthompson** [reply]
Thank you!!

**Reply by vladdoster** [reply]
Thanks @sm1999!
